packages/wdio-spec-reporter/src/index.ts

```diff
@@ -69,7 +68,16 @@ export default class SpecReporter extends WDIOReporter {
                 return this._originalStdoutWrite(chunk, encoding, callback)
             }
         }
+        this._chalk = new Chalk(options.color === false ? { level:0 } : {})
+    }
 
+    /**
+     * @param state state of test execution
+     * @param msg the message to print in terminal
+     * @returns colord value based on chalk to print in terminal
+     */
+    setMessageColor(message:string, state = 'default'): string{
```

Comment:
better:
```suggestion
    setMessageColor(message:string, state?: State): string{
```

now `getColor` will automatically return `grey`.

Reply:
resolved